src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private int index = new Random().nextInt(Colors.values().length);
```

Comment:
index should be calculated inside `getRandomColor()` method. `Random` should be `private final` class-level field